### PR TITLE
Add blogpost "Editor Support, Custom Operators and More"

### PR DESCRIPTION
**Pull request by IwanKaramazow**

### CODE REVIEW

**Comment by d4h0** [comment]
> The second reason we’re introducing custom operators is to shorten the time to get developers up and running with the new `.res` syntax with `.re` files. The new `.res` printer offers vivid, crystal clear code. We don’t want you missing out on all the good stuff!

I have read this sentence more than 5 times, and I still don't understand what it means. Maybe it would make sense to reword it a bit.

**Comment by jfrolich** [comment]
Amazing post. And I am stoked about this editor plugin. This will definitely push me over the line to use ReScript.

BTW, you are free to use https://familyfive.app/ as an example of a ReScript codebase. We build an award-winning app (multiple times Apple App of the Day) used by 70,000 families. And we have a ReScript codebase that covers web and a React Native app.

**Comment by IwanKaramazow** [comment]
@jfrolich Thanks for the suggesting! Family5 is all about allowing families spending more quality time together?

@d4h0 good point, will rephrase.

**Comment by jfrolich** [comment]
> @jfrolich Thanks for the suggesting! Family5 is all about allowing families to spend more quality time together?

Yes. We offer fun development-boosting activities you can do with the kids, goal setting, and advice on how to resolve challenging situations. We also offer live workshops via Zoom, that cover certain topics around parenting.

File: _blogposts/2020-11-12-rescript-the-road-ahead.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+In the coming days, Cristiano and Cheng Lou will finally be ready to unveil the new VSCode ReScript experience, including diagnosis, autocompletion, jump to definition and type hints!
+
+<Video withShadow={true} src="https://player.vimeo.com/video/477758754" />
+
+Patrick and Maxim will also help polishing the same feature set for Vim and Sublime Text.
+
+## Compiler & Build
+
+Hongbo continues to fine tune the compiler and build system for performance and stability. There are several improvements coming down the line. Expect your build to keep getting faster and more robust!
+
```
Comment (bobzhang) [inline _blogposts/2020-11-12-rescript-the-road-ahead.mdx:30]: The current priority is focused on the user experience of pinned packages support

**Pull request merged**